deliveries/chainide-polkaholic-integrated-wasm-contract-explorer-integration-milestone_1.md

```diff
@@ -0,0 +1,190 @@
+# Milestone Delivery :mailbox:
+
+
+- **Application Document:** [https://github.com/use-inkubator/Ecosystem-Grants/blob/master/applications/chainide-polkaholic-integrated-wasm-contract-explorer-integration.md](https://github.com/use-inkubator/Ecosystem-Grants/blob/master/applications/chainide-polkaholic-integrated-wasm-contract-explorer-integration.md)
+
+**Context**
+
+[ChainIDE](https://chainide.com/) is a development environment for EVM Contracts being extended to include WASM Contract templates.
+
+[Polkaholic.io](https://polkaholic.io/), developed by Colorful Notion, is a multichain block explorer for the Substrate ecosystem covering 75+ chains.
+
+
+
+
+## Milestone 1 Deliverables
+
+In Summer 2023, ChainIDE and Colorful Notion worked to ChainIDE+Polkaholic.io to support WASM Contract Developers with a fully integrated WASM Contract Development platform in Milestone 1.  Key results from this Milestone are:
+* a complete WASM Contract Development platform from ChainIDE
+* a full set of WASM Contract Templates: Flipper, PSP22, PSP34, UniswapV2, and more
+*  WASM Contract Verification APIs
+*  WASM Contract Developer Dashboards
+*  WASM Contract API from Polkaholic.io
+*  WASM Contract UI from Polkaholic.io
+
+WASM Contract Developer feedback (new features) will be requested in October 2023 from Milestone 1 Deliverables and used to draft a Milestone 2 follow-on proposal.
+| Number | Deliverable | Link | Notes |
+| --- | --- | --- | --- |
+| 0a. | License | [Polkaholic GNU GPL v3](https://github.com/colorfulnotion/polkaholic/blob/main/LICENSE) |   |
+| 0b. | Polkaholic Documentation | See 0b. Polkaholic Documentation below  | See Polkaholic screenshots and URLs |
+| 0c. | Testing and Testing Guide | See 0c. below | See ChainIDE video walkthrough |
+| 1. | Repository | https://github.com/colorfulnotion/polkaholic  |  |
+| 2. | Repository | https://staging-9589904a8a.chainide.com/s/dashboard/projects | ChainIDE is only semi-open source. This is [the open-source plugin template for ChainIDE](https://github.com/WhiteMatrixTech/chainide-plugin-doc). If you need other open-source plugins, please let us know. |
```

Comment:
Can you link to some of the other ink! relevant components of ChainIDE? It's unclear to me what's open-source and what's not

Reply:
Hi, I come from ChainIDE, and as of now, the only open-source component is the open-source plugin template, allowing users to develop plugins on ChainIDE. However, {the other modules](https://chainide.gitbook.io/chainide-english-1/chainide-modules), including the backend, are not open-source. If you need other parts to be open-source as well, I believe we can discuss the possibility further.